### ISSUE TITLE
bazel query is broken when base_workspace is checked in

### ISSUE THREAD

**Issue by mrdomino** [bazelbuild/bazel#809]
My current approach to a fully vendored bazel is to check the base workspace, as well as binaries for every platform, into my repository, and to create appropriate symlinks for tools/ and third_party/.

But a bunch of stuff in base_workspace (at least on 0.1.3) refers to targets that don't exist, e.g.:

```
ERROR: /Users/steven/ix/vendor/bazel/latest/base_workspace/tools/build_rules/rust/BUILD:27:1: no such package '@rust-darwin-x86_64//': error loading package 'external': The repository named '@rust-darwin-x86_64' could not be resolved and referenced by '//vendor/bazel/latest/base_workspace/tools/build_rules/rust:rustc_lib'.
```

So bazel query produces a lot of errors.

**Comment by damienmg** [comment]
You could just check in the binary without the tools that are already
included in the binary itself.

**Comment by mrdomino** [comment]
This seems to work. Why is base_workspace still included in the 0.1.3 package then? Just legacy?

**Comment by damienmg** [comment]
It's legacy because we still needs it for skylark dependency. It is going to be removed in 0.2
